Source/Core/Core/PowerPC/Jit64/Jit.h

```diff
@@ -18,6 +18,8 @@
 // ----------
 #pragma once
 
+#include <rangesizeset.h>
```

Comment:
The externals project should likely be structured so that the include is within an include folder with a CMake file defining a header only target in the externals folder.

e.g.

`include/rangesizeset/rangesizeset.h` (feel free to rename the first `rangesizeset` to a name of preferred choosing if you want).

```cmake
# Other pre-reqs before this

add_library(RangeSizeSet INTERFACE)
target_include_directories(RangeSizeSet INTERFACE include)
```

Then link it into core. This way the include would be:

`<rangesizeset/rangesizeset.h>`

Which significantly reduces the chances of this include ever clashing with anything that ends up being potentially added to externals in the future.

Reply:
I assume this is fine now?